policyengine_canada/variables/gov/provinces/nt/tax/income/credits/living_tax_credit/nt_living_tax_credit.py

```diff
@@ -0,0 +1,19 @@
+from policyengine_canada.model_api import *
```

Comment:
we can switch to just one file with a marginal rate parameter and single amount for max amount